### PR TITLE
fix: Use workflowRun IDs in inner consensusGenomes where, not sequencingRead IDs

### PR DESCRIPTION
**Pull request by bchu1**
# Pull Request

## JIRA Ticket

I was using the wrong IDs to filter `ConsensusGenome`s, therefore wasn't getting any back.

Also add support for query that only fetches `id` (will be used for filters).

Also fix test util assert function that wasn't properly handling single whitespace characters.

## Tests

Should be able to see `ConsensusGenome` fields in samples table in NextGen.

### CODE REVIEW

File: example-queries/sequencing-reads-query-id-fe.graphql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+query DiscoveryViewFCSequencingReadsQuery(
+    $input: queryInput_fedSequencingReads_input_Input
+) {
+    fedSequencingReads(input: $input) {
+        id
+    }
+}
```
Comment (ehoops-cz) [inline example-queries/sequencing-reads-query-id-fe.graphql:7]: Do we care about ending files with a newline?

**Review by ehoops-cz — APPROVED** [review]

File: example-queries/sequencing-reads-query-id-fe.graphql (hunk 1 above)
Comment (bchu1) [reply example-queries/sequencing-reads-query-id-fe.graphql:7]: For these tests it should be ok, we're stripping all whitespace anyways
